fix

  tcp.c: In function 'tcp_update_check_tcp4':
  tcp.c:769:9: error: unknown type name '__sum16'
    769 |         __sum16 *check;
        |         ^~~~~~~
  tcp.c:800:18: error: '__sum16' undeclared (first use in this function)
    800 |         check = (__sum16 *)ptr;
        |                  ^~~~~~~
  tcp.c:800:18: note: each undeclared identifier is reported only once for each function it appears in
  tcp.c:800:27: error: expected expression before ')' token
    800 |         check = (__sum16 *)ptr;
        |                           ^
  tcp.c: In function 'tcp_update_check_tcp6':
  tcp.c:819:9: error: unknown type name '__sum16'
    819 |         __sum16 *check;
        |         ^~~~~~~
  tcp.c:851:18: error: '__sum16' undeclared (first use in this function)
    851 |         check = (__sum16 *)ptr;
        |                  ^~~~~~~
  tcp.c:851:27: error: expected expression before ')' token
    851 |         check = (__sum16 *)ptr;
        |                           ^
  make: *** [Makefile:107: passt] Error 1

--- a/tcp.c
+++ b/tcp.c
@@ -270,6 +270,7 @@
 #include <stdlib.h>
 #include <errno.h>
 #include <limits.h>
+#include <linux/types.h>
 #include <net/ethernet.h>
 #include <net/if.h>
 #include <netinet/in.h>
